bin/dep

```diff
@@ -23,23 +23,23 @@ elif [ "$(uname -o)" = "Msys" ]; then
   exe=.exe
 fi
 
-depbin=.dep${exe}
+depbin="${rootdir}/.dep-${depversion}${exe}"
 depurl="${dep_base_url}dep-${os}-amd64${exe}"
 
 if [ ! -f "$depbin" ]; then
   tmp=$(mktemp -d -t dep.XXX)
   (
     cd "$tmp"
-    curl -L --silent --fail -o "$depbin" "$depurl"
-    ddv="dep_digest_${os}"
-    (echo "${!ddv} *$depbin" | shasum -c -a 256 -p -s -) || {
-      echo Actual digest of $depbin does not match expected digest.
+    curl -L --silent --fail -o depbin "$depurl"
+    sha=$(curl -L --silent --fail "${depurl}.sha256" | awk '{ print $1 }')
+    (echo "$sha *depbin" | shasum -c -a 256 -p -s -) || {
```

Comment:
should this be `$depbin` or perhaps it doesn't matter?

Reply:
It can't be `$depbin` since `$depbin` is now an absolute path, and that won't work here in the tmp dir. So instead I'm using the filename "depbin" everywhere in tmp.